Source/carthage/Build.swift

```diff
@@ -58,6 +59,7 @@ public struct BuildCommand: CommandProtocol {
 				<*> ColorOptions.evaluate(m)
 				<*> m <| Option(key: "verbose", defaultValue: false, usage: "print xcodebuild output inline")
 				<*> m <| Option(key: "project-directory", defaultValue: FileManager.default.currentDirectoryPath, usage: "the directory containing the Carthage project")
+				<*> m <| Option(key: "log-path", defaultValue: BuildCommand.defaultLogPath, usage: "path to the xcode build output. A temporary file is used by default")
```

Comment:
I don't think he's suggesting that we apply mktemps to provided log paths. Just that if you make it an optional `String?`, then at the point where you decide whether to use mktemps, it can become:
`if path == nil {`
instead of
`if path == BuildCommand.defaultLogPath {`